commerce_coordinator/settings/base.py

```diff
@@ -348,7 +348,11 @@ def root(*path_fragments):
 # Needed to link to the payment micro-frontend
 PAYMENT_MICROFRONTEND_URL = 'replace-me'
 
-# Coordinator filters should NEVER be allowed to fail silently
+# Checkout view urls
+COMMERCETOOLS_FRONTEND_URL = 'https://commerce_tools_frontend_url'
+FRONTEND_APP_PAYMENT_URL = 'https://payment_page_url'
+
+# Filters PoC
```

Comment:
Can you check if this comment is required?

Reply:
I'm not too sure. @grmartin could you comment on this?